contrib/hack/test_walkthrough.sh

```diff
@@ -135,6 +135,10 @@ retry -n 10 \
 
 echo 'Waiting on pods to come up...'
 
+wait_for_expected_output -x -e 'Pending' -n 10 \
```

Comment:
@pmorie It's already doing that; this and the `ContainerCreating` status checks are just to wait until we're in a non-creating/pending state, and then upon reaching that on line 146 we check if the state we ended up in is actually `Ready`.